### ISSUE TITLE
Performance dashboard should use same color:language correspondence for entire dashboard (possibly all dashboards?)

### ISSUE THREAD

**Issue by nathanielmanistaatgoogle** [grpc/grpc#13986]
On https://performance-dot-grpc-testing.appspot.com/explore?dashboard=5652536396611584 the color with which Python is encoded on each chart is inconsistent, making it hard to reason from one chart to the next about a single language.

The color-to-language correspondences on the page should be consistent across all charts.

It should probably also be the case that the color-to-language correspondences should be consistent across all dashboards.

Python calls dibs on teal!

**Comment by ctiller** [comment]
This should be a simple matter of making sure the languages are queried in the same order... simply rewrite every SQL query in a consistent fashion and be done!

**Comment by nathanielmanistaatgoogle** [comment]
@ctiller: not all languages appear on all charts; does that affect the problem?

**Comment by matt-kwong** [comment]
I've tried to make the languages consistent across the charts for this dashboard, except where the chart only has one language. I left the colors default there. 

Unfortunately, I couldn't honor the request for Python to have teal. I chose colors associated with each language, and Go makes a really strong case for teal with the Go Gopher. I gave Python yellow for the Python logo. 

I'm not sure which other dashboards are actively being looked at, but if there are any, I can update those also.
